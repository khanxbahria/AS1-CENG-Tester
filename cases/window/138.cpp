#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
Tab M = Tab("c", "lll","k");
wind->getActiveTab();
Tab S = Tab("kk", "sss","bbb");
wind->changeActiveTabTo(4);
Tab Y = Tab("xxx", "n","hhh");
wind->closeTab();
Tab T = Tab("k", "www","aa");
wind->moveActiveTabTo(1);
Tab X = Tab("vvv", "aa","vv");
wind->closeTab();
wind->print();
Tab V = Tab("aa", "ff","e");
wind->print();
Tab J = Tab("ppp", "aa","ddd");
wind->closeTab();
Tab P = Tab("uu", "vvv","rr");
wind->moveActiveTabTo(1);
wind->newTab(J);
Tab W = Tab("gg", "jjj","oo");
wind->isEmpty();
Tab Q = Tab("tt", "bb","ccc");
wind->getActiveTab();
Tab D = Tab("u", "i","aaa");
wind->newTab(D);
Tab G = Tab("dd", "uuu","aa");
wind->newTab(G);
Tab I = Tab("jj", "ooo","ii");
wind->closeTab();
Tab Z = Tab("rrr", "z","gg");
wind->print();
wind->closeTab();
wind->isEmpty();
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->newTab(I);
Tab K = Tab("f", "xxx","s");
wind->moveActiveTabTo(4);
Tab H = Tab("i", "g","r");
wind->isEmpty();
wind->newTab(H);
wind->isEmpty();
wind->closeTab();
Tab U = Tab("ddd", "kkk","m");
wind->closeTab();
wind->newTab(K);
wind->isEmpty();
wind->closeTab();
wind->newTab(M);
wind->getActiveTab();
Tab N = Tab("l", "ooo","ee");
wind->changeActiveTabTo(4);
Tab L = Tab("v", "gg","ddd");
wind->changeActiveTabTo(4);
wind->newTab(L);
wind->moveActiveTabTo(4);
wind->isEmpty();
wind->moveActiveTabTo(2);
wind->isEmpty();
Tab B = Tab("e", "a","j");
wind->print();
wind->getActiveTab();
wind->newTab(B);
wind->print();
Tab O = Tab("e", "d","a");
wind->print();
Tab C = Tab("ee", "hh","fff");
wind->getActiveTab();
Tab R = Tab("jj", "jj","s");
wind->print();
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->changeActiveTabTo(5);
wind->print();
wind->moveActiveTabTo(1);
wind->print();
wind->isEmpty();
wind->newTab(C);
Tab F = Tab("rr", "zz","e");
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(0);
wind->changeActiveTabTo(5);
wind->isEmpty();
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(1);
wind->print();
wind->getActiveTab();
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->isEmpty();
wind->closeTab();
wind->getActiveTab();
wind->newTab(F);
wind->print();
wind->changeActiveTabTo(2);
wind->changeActiveTabTo(4);
wind->changeActiveTabTo(5);
wind->closeTab();
wind->newTab(N);
wind->isEmpty();
wind->getActiveTab();
wind->closeTab();
wind->newTab(O);
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(4);
wind->moveActiveTabTo(1);
wind->print();
wind->print();
wind->getActiveTab();
wind->moveActiveTabTo(3);
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->newTab(P);
wind->print();
Tab A = Tab("jjj", "bbb","ttt");
wind->getActiveTab();
wind->newTab(A);
wind->moveActiveTabTo(5);
wind->getActiveTab();
wind->getActiveTab();
Tab E = Tab("uu", "www","ttt");
wind->print();
wind->newTab(E);
wind->getActiveTab();
delete wind;
}
	return 0;
}