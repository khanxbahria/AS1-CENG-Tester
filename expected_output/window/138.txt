Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 1
aa
ppp
ddd

i
u
aaa

The active tab is tab 2
aa
ppp
ddd

lll
c
k

gg
v
ddd

The active tab is tab 3
aa
ppp
ddd

lll
c
k

gg
v
ddd

a
e
j

The active tab is tab 3
aa
ppp
ddd

lll
c
k

gg
v
ddd

a
e
j

The active tab is tab 3
aa
ppp
ddd

lll
c
k

gg
v
ddd

a
e
j

The active tab is tab 0
aa
ppp
ddd

lll
c
k

gg
v
ddd

a
e
j

The active tab is tab 1
lll
c
k

aa
ppp
ddd

gg
v
ddd

a
e
j

The active tab is tab 1
aa
ppp
ddd

lll
c
k

hh
ee
fff

gg
v
ddd

a
e
j

The active tab is tab 2
aa
ppp
ddd

hh
ee
fff

zz
rr
e

gg
v
ddd

a
e
j

The active tab is tab 1
aa
ppp
ddd

d
e
a

hh
ee
fff

zz
rr
e

gg
v
ddd

The active tab is tab 1
aa
ppp
ddd

d
e
a

hh
ee
fff

zz
rr
e

gg
v
ddd

The active tab is tab 3
aa
ppp
ddd

hh
ee
fff

zz
rr
e

vvv
uu
rr

The active tab is tab 4
aa
ppp
ddd

hh
ee
fff

zz
rr
e

vvv
uu
rr

bbb
jjj
ttt

